### PR TITLE
ropper: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 1.13.8**

### CODE REVIEW

File: pages/common/ropper.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`ropper --file {{path/to/binary}} --type {{rop|job|sys|all}}`
+
+- List gadgets in the binary not including bad bytes:
+
+`ropper --file {{path/to/binary}} --badbytes {{byte_string}}`
+
+- List gadgets in the binary with up to given instructions count:
+
+`ropper --file {{path/to/binary}} --inst-count {{count}}`
```
Comment (kbdharun) [inline pages/common/ropper.md:24]:
```suggestion
- List gadgets in the binary file:

`ropper --file {{path/to/binary}}`

- List gadgets in the binary file, filtered by regular expression:

`ropper --file {{path/to/binary}} --search {{regex}}`

- List gadgets in the binary file of the specified type:

`ropper --file {{path/to/binary}} --type {{rop|job|sys|all}}`

- List gadgets in the binary file, not including bad bytes:

`ropper --file {{path/to/binary}} --badbytes {{byte_string}}`

- List gadgets in the binary file, with up to the given instructions count:

`ropper --file {{path/to/binary}} --inst-count {{count}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages/common/ropper.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# ropper
+
+> Find ROP gadgets in binary files.
+> More information: <http://scoding.de/ropper/>.
+
+- List gadgets in the binary file:
+
+`ropper --file {{path/to/binary}}`
+
+- List gadgets in the binary file, filtered by regular expression:
```
Comment (mfrw) [inline pages/common/ropper.md:10]: Teeny nit: [same thing without a `,`] Its your call if you want to take it :)
```suggestion
- Filter gadgets in the binary file by a regular expression:
```

File: pages/common/ropper.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`ropper --file {{path/to/binary}} --search {{regex}}`
+
+- List gadgets in the binary file of the specified type:
```
Comment (mfrw) [inline pages/common/ropper.md:14]:
```suggestion
- List gadgets of specified type in the binary file:
```

File: pages/common/ropper.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`ropper --file {{path/to/binary}} --type {{rop|job|sys|all}}`
+
+- List gadgets in the binary file, not including bad bytes:
```
Comment (mfrw) [inline pages/common/ropper.md:18]:
```suggestion
- Exclude bad byte gadgets in the binary file:
```

File: pages/common/ropper.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`ropper --file {{path/to/binary}} --badbytes {{byte_string}}`
+
+- List gadgets in the binary file, with up to the given instructions count:
```
Comment (mfrw) [inline pages/common/ropper.md:22]:
```suggestion
- List gadgets up to the specified instruction count in the binary file:
```

**Review by mfrw — COMMENTED** [review]
Slight Cosmetic cleanup :)

**Review by mfrw — APPROVED** [review]
Thank you @johandegn for your contribution.

LGTM :)

**Pull request merged**